plugins/aea-ledger-fetchai/aea_ledger_fetchai/_cosmos.py

```diff
@@ -1125,26 +1098,37 @@ def send_signed_transaction(self, tx_signed: JSONLike) -> Optional[str]:
         :param tx_signed: the signed transaction
         :return: tx_digest, if present
         """
-        if self.is_cosmwasm_transaction(tx_signed):
-            tx_digest = self._try_execute_wasm_transaction(tx_signed)
-        elif self.is_transfer_transaction(tx_signed):
-            tx_digest = self._try_send_signed_transaction(tx_signed)
-        else:  # pragma: nocover
-            _default_logger.warning(
-                "Cannot send transaction. Unknown transaction type: {}".format(
-                    tx_signed
-                )
-            )
+
```

Comment:
This method used to handle both cosmwasm and signed TXs. Now it only appears to handle signed TXs. What happened to the other case?

Reply:
Method `_try_send_signed_transaction` does not seem to be used anymore.